player/score_bounded_mcts/score_bounded_mcts_player.py

```diff
@@ -0,0 +1,11 @@
+from player.mcts.mcts_configuration import MctsConfiguration
+from player.mcts.mcts_player import MctsPlayer
+from player.score_bounded_mcts.score_bounded_mcts_node import ScoreBoundedMctsNode
+from player.mcts.imcts_node import IMctsNode
+
+
+class ScoreBoundedMctsPlayer(MctsPlayer):
+    node_type: IMctsNode = ScoreBoundedMctsNode
```

Comment:
I guess here you should rather override method for creating tree root using `ScoreBoundedMctsNode`. Similar to how it is done for `HistoryHeuristicPlayer`.